pyg_lib/csrc/ops/matmul.cpp

```diff
@@ -57,9 +132,10 @@ class SegmentMatmul : public torch::autograd::Function<SegmentMatmul> {
       input_grad = _segment_matmul(grad_out, ptr, other_t);
     }
     if (torch::autograd::any_variable_requires_grad({other})) {
-      // TODO (matthias) implement backward pass for `other`.
+      variable_list grad_out_list = break_w_ptr(grad_out, ptr);
+      variable_list other_list = break_w_ptr(other, ptr);
+      other_grad = reflatten(_grouped_matmul(grad_out_list, other_list));
```

Comment:
This will still fail since it will not pick up the right dispatcher here, correct?